qa/rpc-tests/wallet.py

```diff
@@ -56,8 +56,10 @@ def run_test (self):
         assert_equal(len(self.nodes[2].listunspent()), 0)
 
         # Send 21 BTC from 0 to 2 using sendtoaddress call.
-        self.nodes[0].sendtoaddress(self.nodes[2].getnewaddress(), 11)
+        addr = self.nodes[2].getnewaddress()
+        self.nodes[0].sendtoaddress(addr, 11)
```

Comment:
Can you please fix the comment above? You are sending 11 BTC but the comment says 21 BTC.